dart/gui/osg/SupportPolygonVisual.cpp

```diff
@@ -387,8 +386,7 @@ void SupportPolygonVisual::initialize()
         dart::dynamics::Frame::World(), "com");
 
   mComRadius = mCentroidRadius;
-  mCom->setShape(std::make_shared<dart::dynamics::EllipsoidShape>(
-                                mComRadius/2.0*Eigen::Vector3d::Ones()));
```

Comment:
@mxgrey Did you mean `mComRadius*2.0` in the first place? In that case, the below code should be just `mComRadius` rather than `mComRadius/4.0`.